certified-connectors/AXtension Content Gate/apiDefinition.swagger.json

```diff
@@ -15,655 +15,370 @@
   "schemes": [
     "https"
   ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
   "paths": {
-    "/api/ContentEntityProperties/schema": {
+    "/api/businessentities/{providerReferenceId}/{externalType}/{externalId}/Requirements": {
       "get": {
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "id"
+                  },
+                  "contentEntityTemplateId": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "contentEntityTemplateId"
```

Comment:
@JoepKillaars All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Applies the same comment to all the similar description values.